code/controllers/subsystem/weather.dm

```diff
@@ -13,7 +13,8 @@ SUBSYSTEM_DEF(weather)
 		var/datum/weather/W = V
 		if(W.aesthetic)
 			continue
-		for(var/mob/living/L in GLOB.mob_list)
+		for(var/i in GLOB.living_mob_list)
+			var/mob/living/L = i
```

Comment:
living_mob_list is stupid and I hate, it only contains /mob/livings that have stat != DEAD. We really need a seperate list for /mob/living or rename the type to something more logical. Anyway, revert this

Reply:
This was supposed to be the second comment in the first review which is why it doesn't read well